index.html

```diff
@@ -1,57 +1,226 @@
-
-    <form id="survey-form">
-        <label for="name" id="name-label">name:</label>
-        <input type="text" id="name" placeholder="Enter your name" />
-        <label for="email" id="email-label">email:</label>
-        <input type="Email" id="email" placeholder="Enter your email" />
-        <label for="number" id="number-label">number"</label>
-        <input type="Number" id="Number" placeholder="Enter your number" min="10" max="100" />
-
-        <label for="drop-down">Select an option:</label>
-        <select id="dropdown">
-        <option value="fruit1">Apples</option>
-        <option value="fruit2">Bananas</option>
-        <option value="fruit3">Melons</option>
-        <option value="fruit4">Paw-paws</option>
-      </select>
-        <label for="Green Apples">Green Apples
-		<input id="Green Apples" type="radio" name="Green Apples" value="Green Apples">
-	</label>
-        <label for="Red Apples">Red Apples
-		<input id="Red Apples" type="radio" name="Green Apples" value="Red Apples">
-    </label>
-
-        <label for="box1">Box1
-      <input id="box1" type="checkbox" name="box1" value="box1">
-    </label>
-        <label for="box2">Box2
-      <input id="box2" type="checkbox" name="box2" value="box2">
-    </form>
-
-    <div id= "comment-section">
-      <label for="comments">
-        Do you have any Comments?</label>
-
-        <textarea id="comments" class="input-field" style="height:50px;resize:vertical;" name="comment" placeholder="Enter your comment here..."></textarea>
-        </div>
-        <div><button type="submit" id="submit" class="dropdown"> Submit </button></div>
-
-        <script src="js/index.js"></script>
-</body>
-
+  <head>
+    <meta charset="utf-8">
+    <title>Home</title>
+    <link href="css/bootstrap.css" rel="stylesheet">
+    <link href="css.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center">Meditation app survey form</h1>
+        <p id="description" class="description text-center">
+          Thank you for taking the time to help us improve our app
+        </p>
+      </header>
+      <form id="survey-form">
+        <div class="form-group">
+          <label id="name-label" for="name">Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="form-control"
+            placeholder="Enter your name"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="email-label" for="email">Email</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="form-control"
+            placeholder="Enter your Email"
+            required
+          />
+        </div>
+        <div class="form-group">
+          <label id="number-label" for="number"
+            >Age<span class="clue">(optional)</span></label
```

Comment:
Kindly close the tags within one line if possible to enhance a good visual appeal when viewers are going through your code. Example: `<label id="id-name" for="purpose">GPA<span class="class-name">4.0</span></label>`